cmd/transfer.go

```diff
@@ -75,6 +77,9 @@ func Transfer(ctx context.Context, rpcAddress string, chainID int64, mnemonic, i
 		return err
 	}
 	log.Printf("1. token approve success (TxHash: %s)\n", tx.Hash().Hex())
+	if err := waitAndCheckStatus(ctx, chainA, tx); err != nil {
```

Comment:
nit: Since `chainB` does not exist in the function, can we simply use `chain` instead?

Reply:
Rename chainA -> chain
https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/a282bb6171e91448a6026fa7a2ef440e62553f08